hw/ip/rv_plic/fpv/rv_plic_component_fpv.core

```diff
@@ -1,24 +0,0 @@
-CAPI=2:
-# Copyright lowRISC contributors.
-# Licensed under the Apache License, Version 2.0, see LICENSE for details.
-# SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:fpv:rv_plic_component_fpv:0.1"
-description: "FPV testbench for RISC-V PLIC example"
-
-filesets:
-  files_fpv:
-    depend:
-      - lowrisc:ip:tlul
-      - lowrisc:prim:all
-    files:
-      - vip/rv_plic_assert_fpv.sv
```

Comment:
this VIP file is missing now in the FPV build. either we include the `lowrisc:fpv:rv_plic_component_fpv` core again, or we add the `vip/rv_plic_assert_fpv.sv` file to the `rv_plic_fpv.core`.

Reply:
Good catch, I added the file to `hw/top_earlgrey/ip_autogen/rv_plic/fpv/rv_plic_fpv.core`.